.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```
@@ -0,0 +1,69 @@
+name: Bug Report
+description: Create a report to help us improve
+title: " "
+labels: ['type:Bug', 'status:Unconfirmed']
+body:
+  - type: textarea
+    id: "summary"
+    attributes:
+      label: Issue summary
+      description: "A brief summary of the issue"
+    validations:
+      required: true
+  - type: textarea
+    id: "steps"
+    attributes:
+      label: "Steps to reproduce"
+      value: |
+        1. Step 1
+        2. Step 2
+        3. ...
+    validations:
+      required: true
+  - type: input
+    id: "version-pyth"
+    attributes:
+      label: "Python version"
+      description: "Execute python --version"
+      placeholder: "Execute python --version"
+    validations:
+      required: true
+  - type: input
+    id: "version-django"
+    attributes:
+      label: "Django version"
+      description: "Execute pip show django \| grep Version"
+      placeholder: "Execute pip show django \| grep Version"
```

We don't need the escape character here. 
```suggestion
      placeholder: "Execute pip show django | grep Version"
```